src/parser/view_transform.c

```diff
@@ -1651,6 +1651,36 @@ mq_substitute_spec_in_method_and_hints (PARSER_CONTEXT * parser, PT_NODE * node,
   return node;
 }
 
+/*
+ * check the name nodes from view spec in "on duplicate key update" clause
+ */
+static PT_NODE *
+pt_check_odku_has_vclass (PARSER_CONTEXT * parser, PT_NODE * odku, void *arg, int *continue_walk)
+{
+  PT_NODE *sel_spec = (PT_NODE *) arg;
+  PT_NODE *entity, *class_ = NULL;
+
+  *continue_walk = PT_CONTINUE_WALK;
+
+  if (odku == NULL || odku->node_type != PT_NAME)
+    {
+      return odku;
+    }
+
+  entity = pt_find_spec (parser, sel_spec, odku);
+  if (entity)
+    {
+      class_ = entity->info.spec.flat_entity_list;
+
+      if (class_ && mq_translatable_class (parser, class_))
+	{
+	  class_->info.name.flag |= PT_NAME_INFO_REFERENCED_AT_ODKU;
```

Comment:
준비된 매크로가 있으니 활용하면 좋겠습니다.

```suggestion
	  PT_NAME_INFO_SET_FLAG(class_, PT_NAME_INFO_REFERENCED_AT_ODKU);
```

Reply:
네, 적용했습니다.